### PR TITLE
DO NOT MERGE UNTIL 1.5.0 - Release Notes

### PR DESCRIPTION
**Pull request by susanmdouglas-aws**
Initial check in of release notes for 1.5.0

Signed-off-by: susanmdouglas <susandou@amazon.com>

### Description
Initial check in of release notes for 1.5.0
 
### Issues Resolved
Initial check in of release notes for 1.5.0

### Check List
- [x] Commits are signed per the DCO using `--signoff`

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

**Review by kuntalghosh — APPROVED** [review]

**Review by shardgupta — APPROVED** [review]

File: _includes/releases/babelfish-1.5.0.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+After the extensions are updated, you can check the extension version from the PostgreSQL port:
+
+```bash
+babelfish_db[124386]# \dx
+                                List of installed extensions
+        Name        | Version |   Schema   |                   Description
+--------------------+---------+------------+-------------------------------------------------
+ babelfishpg_common | 1.2.1   | public     | Transact SQL Datatype Support
+                                 ....
+ babelfishpg_tsql   | 1.2.1   | public     | Transact SQL Compatibility
```
Comment (kuntalghosh) [inline _includes/releases/babelfish-1.5.0.md:50]: The extension version is 1.3.0 in 1.5-stable - https://github.com/babelfish-for-postgresql/babelfish_extensions/blob/BABEL_1_5_STABLE/contrib/babelfishpg_tsql/Version.config

Comment (susanmdouglas-aws) [reply _includes/releases/babelfish-1.5.0.md:50]: Thanks Kuntal!  That must have been overlooked in the release notes @suprio-amzn shared...

File: _includes/releases/babelfish-1.5.0.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+After the extensions are updated, you can check the extension version from the PostgreSQL port:
+
+```bash
+babelfish_db[124386]# \dx
+                                List of installed extensions
+        Name        | Version |   Schema   |                   Description
+--------------------+---------+------------+-------------------------------------------------
+ babelfishpg_common | 1.3.0   | public     | Transact SQL Datatype Support
+                                 ....
+ babelfishpg_tsql   | 1.3.0   | public     | Transact SQL Compatibility
```
Comment (kuntalghosh) [inline _includes/releases/babelfish-1.5.0.md:50]: babelfishpg_common - 1.2.1
babelfishpg_tsql - 1.3.0

Comment (susanmdouglas-aws) [reply _includes/releases/babelfish-1.5.0.md]: Updated...

**Review by kuntalghosh — APPROVED** [review]

**Review by suprio-amzn — APPROVED** [review]
Latest revision looks good.

**Pull request merged**